localisation/english/btr_buildings_l_english.yml

```diff
@@ -2,7 +2,7 @@
 
  ### Asari Republic Buildings ###
  
- btr_building_asrep_comm_center:0 "Republican Communication Center"
+ btr_building_asrep_comm_center:0 "Communication Centre of the Republics"
```

Comment:
@puckhead73 Can you confirm this change? Its English just seems weird to me.

Reply:
@puckhead73 feel free if you think it should be different this is just what Mis provided after we pointed out the Republican thing.